### ISSUE TITLE
Creating a new page using new nasa apis 

### ISSUE THREAD

**Issue by surajvast1** [PranavBarthwal/cosmoXplore#383]
### Describe the feature

We are looking to create a new page on our website that matches our current theme and implements exciting new features using NASA APIs. Our goal is to enhance user engagement with interactive and educational content. We are open to various ideas and would love to see what the community can come up with.

Ideas:
One potential feature is an Earth View using latitude and longitude coordinates, leveraging the NASA Earth API to provide real-time satellite imagery of any location on Earth. However, we are open to any innovative suggestions that utilize the rich data provided by NASA APIs.

checkout this link for Nasa Open Api links :: https://api.nasa.gov/#browseAPI

### Would you like to work on this issue?

None

**Comment by DonaldReddy** [comment]
@surajvast1 for your idea, do you want to show the earth view and map view based on the user's current locations?

**Comment by surajvast1** [comment]
Yeah this can be implemented where we fetch location and on the basic of that it calculates latitude and longitude and shows the image .
this could be done with some free apis available.

**Comment by DonaldReddy** [comment]
@surajvast1  i can create a page to show earth view of the user current location and user can type lat and long coordinates to fetch earth view of their choice. if it sounds good ?,ill raise an issue.

**Comment by surajvast1** [comment]
Yeahhhh go onnn

**Comment by surajvast1** [comment]
Don't give user to type latitude and longitude if its posssible we can use a api of googe or any other free apis where people just enters their location and they get the image by automatic calculations of lat and lon

**Comment by DonaldReddy** [comment]
Ok so l, I'll implement the following features 

1. Showing earth image by taking user current location 

2. Showing earth image by taking place name from user.

**Comment by surajvast1** [comment]
No i guess when we take location both will be covered

**Comment by DonaldReddy** [comment]
Ok so just taking the place name and showing them Earth view.

**Comment by ShivangiNegi00** [comment]
I would like to work on this issue, please assign this issue to me .

**Comment by Prince2301p** [comment]
Is there an existing issue for this?

I have searched the existing issues
New Tool Description
Slite can help you keep up with your work. This app serves as a knowledge base where you can store your business’s collective brainpower.

Use Case
Slite can help you keep up with your work. This app serves as a knowledge base where you can store your business’s collective brainpower.

Benefits
No response

Add ScreenShots
No response

Priority
High

Record

I have read the Contributing Guidelines

I'm a GSSOC'24 contributor

I have starred the repository
